.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,300 @@
+name: Build LLDAP Bin and Dockerize
```

Comment:
Maybe just "Docker" for the name? That's clear enough and lets you see the arch more easily in the checks.